h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/XGBoostTest.java

```diff
@@ -24,6 +25,7 @@
 import java.io.PrintWriter;
 import java.util.Arrays;
 import java.util.HashMap;
+import java.util.Map;
 
 import static water.util.FileUtils.locateFile;
 
```

Comment:
So yeah - this is an old test that had to be updated. It's not really a test for distributed setting. That being said - good point, I should check if any of the XGBoost tests are running in our multi node tests.